cliquet/utils.py

```diff
@@ -208,3 +209,23 @@ def build_response(response, request):
     dict_obj['body'] = body
 
     return dict_obj
+
+
+def encode_header(value, encoding='utf-8'):
+    """Make sure the value is of type ``str`` in both PY2 and PY3."""
+    value_type = type(value)
+    if value_type != str:
+        # Test for Python3
+        if value_type == six.binary_type:  # pragma: no cover
+            value = value.decode(encoding)  # pragma: no cover
```

Comment:
both pragmas are required ? (I would have thought that only the one for ``if`` is necessary)